### ISSUE TITLE
Pretrained checkpoint

### ISSUE THREAD

**Issue by KIMGEONUNG** [CompVis/stable-diffusion#49]
Hi, first of all, thank you for your invaluable work and sharing.
A weak ago, I requested the pretrained model following your discriptions.
However, I only got a mail (below), and I have not received anything about the checkpoint. 

So, my question is
Did you stop the public sharing of the pretrained model?
or
How much time do I wait, more?

Thank you.

![image](https://user-images.githubusercontent.com/32098205/185819776-abc6d19a-0964-4fb3-b512-095a78b5ed12.png)

**Comment by aravind598** [comment]
Hi I am also facing the same problem, I have also requested access to the model for academic research purposes. I have the same questions as OP.

**Comment by leszekhanusz** [comment]
According to [diffusers PR #228](https://github.com/huggingface/diffusers/issues/228), the model will be made public today.

**Comment by aravind598** [comment]
I see but the wont the model include aggressive filters that may distort the output of the model despite the usage of "certain" words in the prompt?

**Comment by israelraizer** [comment]
> I see but the wont the model include aggressive filters that may distort the output of the model despite the usage of "certain" words in the prompt?

No, the filter was only for the discord bot and the website, for the release it will be optional

**Comment by nerdyrodent** [comment]
2pm EST for a public model! #hype

**Comment by bibyts** [comment]
Hi. How to download model.ckpt?

**Comment by parsec501** [comment]
> Hi. How to download model.ckpt?

https://www.reddit.com/r/StableDiffusion/comments/wv4sqt/

**Comment by breadbrowser** [comment]
> > I see but the wont the model include aggressive filters that may distort the output of the model despite the usage of "certain" words in the prompt?
> 
> No, the filter was only for the discord bot and the website, for the release it will be optional

the demo is here https://huggingface.co/spaces/stabilityai/stable-diffusion
